src/components/FilterBox/FilterModuleSmall.css

```diff
@@ -1,58 +0,0 @@
-.Module {
-    position: absolute;
-    left: 110%;
-    width: 250px;
-    padding: 30px;
-    background: #434058;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex-direction: column;
-}
-
-/*Checkboxes styles*/
-input[type="checkbox"] {
-    display: none;
-}
-
-input[type="checkbox"] + label {
```

Comment:
Noch nie gesehen. Man lernt doch nie aus.

Reply:
Hab nochmal eine Karte in der Projektübersicht dafür angelegt, wir gehen die Elemente nochmal durch und verbessern ggf.

Reply:
Also was Reponsiveness angeht